pyproject.toml

```diff
@@ -48,14 +48,31 @@ changelog = "https://github.com/neo4j-contrib/neomodel/releases"
 [project.optional-dependencies]
 dev = [
     "pytest>=7.1",
+    "pytest-cov>=4.0",
     "pre-commit",
+    "black",
+    "isort",
     "Shapely>=1.8.1,<1.9"
 ]
 
 [tool.pytest.ini_options]
-addopts = "--resetdb"
+addopts = "--resetdb --cov=neomodel --cov-report=html:coverage_report"
 testpaths = "test"
 
+[tool.isort]
+profile = 'black'
+src_paths = ['neomodel']
+
+[tool.pylint.'MESSAGES CONTROL']
+disable = 'missing-module-docstring,redefined-builtin,missing-class-docstring,missing-function-docstring,consider-using-f-string,line-too-long'
```

Comment:
Add pylint ignores ; some are temporary to fix issues step by step (like f-string or line-too-long)